### ISSUE TITLE
No-tabs console toolbar always visible to screen reader

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#12825]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 2023.05.0-daily+52
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

When there are other tabs in the Console region (e.g. Terminal), it looks like screenshot below, with the Console toolbar (showing the R version, the working directory, and the clear console button) below the tabs themselves.

<img width="340" alt="Screenshot of Console when there are other tabs" src="https://user-images.githubusercontent.com/10569626/224754544-a8cacb9d-31f2-46df-9eaa-a7c247c6d241.png">

When all other tabs are closed the tabs go away, and the Console toolbar is displayed on the same row as the Minimize/Maximize widgets:

<img width="342" alt="Screenshot of Console when there are no other tabs" src="https://user-images.githubusercontent.com/10569626/224755010-8d068100-3294-4c39-b348-6ff038f9b843.png">

Attempt to navigate the Console region with a screen reader. Start at the Console prompt and move backwards. Try this in both modes (tabs and no tabs).

### Describe the problem in detail

Problem One:

These two toolbars are separate instances, and the "no tabs" one remains visible to a screen reader even when the other one is visible. This makes navigation very confusing.

Problem Two:

When there are no tabs, the Console region continues to have the tabpanel role, which is not semantically accurate, and seems to confuse VoiceOver on macOS (I haven't tried other screen readers). It manifests as the screen reader cursor bouncing around when attempting to navigate backwards through the console (e.g. starting with focus at the Console prompt).

Problem Three:

The language icon (R, Python) in the toolbar has no accessible text and is just read as "image".

Problem Four:

The visual separator ("dot") between the language info and the current working directory is visible to screen reader, but shouldn't be.

### Describe the behavior you expected

- the visually hidden no-tabs toolbar needs to be hidden from screen reader as well
- the tabpanel role and label should only be applied when there are tabs
- the language icons should either be cosmetic (`alt=""`) or given meaningful alt-text
- the dot separator should be aria-hidden

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by jonvanausdeln** [comment]
Verified on 2023.05.0-daily+180
Windows 11, MacOS 12.6

Screen readers seem to behave as expected in the scenarios.
